sdk/python/feast/feature_store.py

```diff
@@ -1508,10 +1508,10 @@ def _read_from_online_store(
 
         # Each row is a set of features for a given entity key. We only need to convert
         # the data to Protobuf once.
-        row_ts_proto = Timestamp()
```

Comment:
since the same object is referenced in the list for each entity row then the last assignment is what gets set for all rows so we need a new proto object for each row